docs/CONTRIBUTING.md

```diff
@@ -23,9 +24,12 @@ you'd like to address unless the proposed fix is minor.
 
 In your Pull Requests (PR), link to the issue that the PR solves.
 
-Please ensure that the base of your PR is the **master** branch. (Note: The default
-GitHub branch is the latest release branch, though you should point all of your changes to be merged into
-master).
+Please ensure that the base of your PR is the **main** branch.
+
+### Commits
+
+We prefer a clean commit history. Please clean up your commits before marking your PR
```

Comment:
What does "clean up your commits" mean int his context? Does it mean that we should make sure to squash all commits before asking for review (cleanup, squash, force-push), or does it just mean to make sure that debug printf's etc are removed (possibly as an additional commit)?

Reply:
Good question! It means we should squash all commits before asking for review. I'll add that sentence!

Reply:
Updated!